ci/test.sh

```diff
@@ -32,5 +32,8 @@ fi
 
 env="$1"
 
+# Activate the tests venv
+source /venv/bin/activate
```

Comment:
This makes sense if we installed the requirements in the virtual env.

I wonder how the tests passed before this change?

Reply:
I guess we haven't tested the pipelines since the update.
Crearly, I forgot to update this project, as I was doing the same for `vdsm` in parallel.